### PR TITLE
Handle www.wikia.com on staging environments

### PR DESCRIPTION
**Pull request by sczerwinski-wikia**
dev: http://www.sczerwinski.wikia-dev.pl/WAM
preview (newStagingUrlFormat=false): http://preview.www.wikia.com/WAM
preview (newStagingUrlFormat=true): http://www.preview.wikia.com/WAM
staging: http://www.wikia-staging.com/WAM
prod: http://www.wikia.com/WAM

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java [hunk 1]
```diff
@@ -133,10 +133,14 @@ public String getWikiGlobalURL(){
     switch (env) {
       case DEV: {
         String devBoxOwner = this.env.split("-")[1];
-        return String.format(getUrlProtocol() + "%s.www.%s", devBoxOwner, env.getWikiaDomain());
+        return String.format(getUrlProtocol() + "www.%s.%s", devBoxOwner, env.getWikiaDomain());
       }
       case SANDBOX: {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java]: MAJOR Extract this nested code block into a method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
MAJOR Reduce this switch case number of lines from 7 to at most 5, for example by extracting code into methods. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 4 major

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [UrlBuilder.java#L101](https://github.com/Wikia/selenium-tests/blob/f05950d4a22c7c0a1a82d1fd63fc72ab6a61923e/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L101): Extract this nested code block into a method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
1. ![MAJOR][MAJOR] [UrlBuilder.java#L111](https://github.com/Wikia/selenium-tests/blob/f05950d4a22c7c0a1a82d1fd63fc72ab6a61923e/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L111): Reduce this switch case number of lines from 7 to at most 5, for example by extracting code into methods. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java [hunk 2]
```diff
@@ -133,10 +133,14 @@ public String getWikiGlobalURL(){
     switch (env) {
       case DEV: {
         String devBoxOwner = this.env.split("-")[1];
-        return String.format(getUrlProtocol() + "%s.www.%s", devBoxOwner, env.getWikiaDomain());
+        return String.format(getUrlProtocol() + "www.%s.%s", devBoxOwner, env.getWikiaDomain());
```
Comment (themech) [inline src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java]: Looks like a long-lasting bug, good catch!

**Comment by WikiaTech** [comment]
SonarQube analysis reported 4 issues
* ![MAJOR][MAJOR] 4 major

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [UrlBuilder.java#L101](https://github.com/Wikia/selenium-tests/blob/0a7d40abff0a99c5e6f14a350e542fbed61a379e/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L101): Extract this nested code block into a method. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1199)
1. ![MAJOR][MAJOR] [UrlBuilder.java#L111](https://github.com/Wikia/selenium-tests/blob/0a7d40abff0a99c5e6f14a350e542fbed61a379e/src/test/java/com/wikia/webdriver/common/core/url/UrlBuilder.java#L111): Reduce this switch case number of lines from 7 to at most 5, for example by extracting code into methods. [rule](http://sonar-s4/coding_rules#rule_key=squid%3AS1151)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**